qiskit/tools/jupyter/version_table.py

```diff
@@ -39,12 +39,9 @@ def qiskit_version_table(self, line="", cell=None):
         packages = []
         qver = qiskit.__qiskit_version__
 
-        packages.append(("Qiskit", qver["qiskit"]))
-        packages.append(("Terra", qver["qiskit-terra"]))
-        packages.append(("Aer", qver["qiskit-aer"]))
-        packages.append(("Ignis", qver["qiskit-ignis"]))
-        packages.append(("Aqua", qver["qiskit-aqua"]))
-        packages.append(("IBM Q Provider", qver["qiskit-ibmq-provider"]))
+        for pkg in qver:
+            if qver[pkg]:
```

Comment:
This will exclude the metapackage if it's not installed. While the earlier version here always included it. I don't have a preference either way, just asking if this was intentional or not.

Reply:
It was intentional.